py/torch_tensorrt/dynamo/runtime/_PythonTorchTensorRTModule.py

```diff
@@ -218,13 +219,25 @@ def set_weight_streaming_budget_v1(
                     self.engine.minimum_weight_streaming_budget
                 )
 
+    def reset_context(self):
```

Comment:
Do these context resets atomically with whatever runtime settting change. Leave as much out of the forward function as we can

Reply:
I came crossed two ideas and tried #1. Please let me know if there is better way to handle it automatically.
1. reset_context(delete context) and  apply set_weight_streaming_budget() api. context is created at forward()
2. Enqueue runtime setting change like set_weight or profile enable. Then delete context->apply pending api-> create context in forward()